cost/terminate_policy/server_terminate.pt

```diff
@@ -0,0 +1,188 @@
+name "Terminate Instances with End Date"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy Template is used to automatically resize AWS Instances based on user-defined standards. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/aws_superseded_instance) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_instances" do
+  label "Permissions for instances"
+  actions "rs_cm.index", "rs_cm.show"
+  resources "rs_cm.instances"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses"
+  description "Email addresses of the recipients you wish to notify"
+end
+
```

Comment:
I believe we decided all policies should have an exclude tag option.    Any reason why this shouldn't have an exclude tag?

Reply:
Do we need to exclude items that have an implicit include?